src/js/S3Client.js

```diff
@@ -237,9 +237,18 @@ export default class S3Client {
         Delete: {
           Objects: objects,
         },
+        BypassGovernanceRetention: true,
       };
       return this.client.deleteObjects(params, (error, data) => {
-        error ? reject(error) : resolve(data);
+        const hasAccessDeniedError =
+          data &&
+          data.Errors &&
+          data.Errors.some(error => error.Code === 'AccessDenied');
```

Comment:
other error should be ignored here ?

Reply:
if we have other errors we reject them directly. 
the problem with this AccessDenied is we get 200 response code from the API, so we have to check manually if we have this error in the data field.